modules/compose/Makefile

```diff
@@ -0,0 +1,44 @@
+DOCKER_NETWORK ?= $(COMPOSE_PROJECT_NAME)_default
+
+DOCKER_MONITOR_TABLE ?= 'table {{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.NetIO}}\t{{.BlockIO}}'
+
+DOCKER_CMD ?= docker run -i --rm --network=$(DOCKER_NETWORK)
+
+SSH_EXEC ?= docker exec -it
+CONTAINER_SHELL ?= sh
+
+DOCKER_COMPOSE ?= docker-compose
+
+## Start local dev environment
+compose/up:
+	@$(DOCKER_COMPOSE) up -d
```

Comment:
unsure about the `-d` as a default behavior. it's misleading as it masks errors. I've paired with customers and they think it's all fine since `compose/up` exists. Yet a lot of errors are masked. I'm unsure since I don't know which use-case to optimize for.